index.php

```diff
@@ -10,6 +10,10 @@
 <?php
 include 'includes/header.php';
 ?>
+
```

Comment:
@pulkitnagpal007 Please add the div over here.
@dheeth 
@sajalbansal2298 
@mahakbansal3

Reply:
Done Sir.